src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write your code here
+const items = document.querySelectorAll('.population');
+const amount = [...items].map(item => item.innerText);
+
+const total = amount.reduce((acc, item) => acc + +item.split(',').join(''), 0);
+const average = total / amount.length;
+
+const totalString = thousandsSeparator(total);
+const averageString = thousandsSeparator(average);
```

Comment:
You can use [Number.toLocaleString()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString)
```suggestion
const totalString = total.toLocaleString('en-US');
const averageString = average.toLocaleString('en-US');
```